### PR TITLE
Fix "you" wording and add notes about repository moves.

### PR DESCRIPTION
**Pull request by parejkoj**

### CODE REVIEW

File: index.rst [hunk 1]
```diff
@@ -157,33 +174,24 @@ Merging in work that had started on the old package
...
-
-Summary of these steps:
-
-1. Ensure all work has moved to new fork
-2. Delete old package
-3. Move new package to `lsst` organization
-4. Update `lsstsw/etc/repos.yaml`
-
-.. _github transfer repository: https://help.github.com/articles/transferring-a-repository-owned-by-your-organization/
+Once you are reasonably confident that there is nobody working on ``meas_worst`` and that all relevant work has been moved to ``pipe_best``, you can delete ``meas_worst`` from `lsst`, and change your ``pipe_best`` remote to point to its final resting place at ``https://github.com/lsst-dm/pipe_best.git``.
```
Comment (ktlim) [inline index.rst]: `meas_worst` should be in `lsst-dm` at this point, and the `pipe_best` remote should be pointed to `lsst`.

Comment (parejkoj) [reply index.rst]: Oops! I thought I fixed that. It's fixed now.

Comment (ktlim) [reply index.rst]: Not quite -- you still have `meas_worst` in `lsst` instead of `lsst-dm`.

Comment (parejkoj) [reply index.rst]: Ah, good catch! Fixed.

File: index.rst [hunk 2]
```diff
@@ -149,6 +150,22 @@ To ensure that other developers are aware of the pending change, post to the app
 
 .. _Community: https://community.lsst.org
 
+Merge to master and move packages on github
+-------------------------------------------
+
+Once your review is complete and you've announced the move as described above, you can merge your branches to master. But merging to master isn't the final step.
+
+All LSST dependencies need to live in the `lsst` github organization. In order to keep our build system happy, you will also need to move ``meas_worst`` and ``pipe_best`` between organizations via the `github transfer repository`_ instructions. Because github puts in an automatic redirect as part of those instructions, people with clones pointing to the pre-move repositories will not have to change their clone `remote` links. You may need the help of a senior developer/manager in order to complete this move step, as the person doing each move needs to be an administrator of both the source and destination organziation.
```
Comment (timj) [inline index.rst]: Two back ticks for the first `lsst` and `remote`? Or even make it a link to the actual github org.

Comment (timj) [reply index.rst]: It's not the build system that cares, it's the system that does releases that cares.

Comment (parejkoj) [reply index.rst]: I've been using single backticks (italics) for organization names throughout.

**Pull request merged**